manifests/opendatahub/rbac/networkpolicy-etcd.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2022 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: etcd
+spec:
+  podSelector:
+    matchLabels:
+      app: model-mesh
```

Comment:
does it need to change to `component: model-mesh-etcd`

Reply:
@rimolive could you confirm which one is the correct one?

Reply:
Should be correct, because this was tested by multiple users

Reply:
@rimolive confirmed the `app: model-mesh` label since it is for model-mesh pods

Reply:
Opened a separate issue for it: https://github.com/opendatahub-io/modelmesh-serving/issues/59